src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,11 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
```

Comment:
The fragment shorthand (<>) is unnecessary here since the div with className 'PostList' is already wrapping the content inside.